src/javascript/binary/pages/bet/tick_trade.js

```diff
@@ -18,7 +18,7 @@ var TickDisplay = function() {
             $self.contract_start_ms = parseInt(data.contract_start * 1000);
             $self.contract_category = data.contract_category;
             $self.set_barrier = ($self.contract_category.match('digits')) ? false : true;
-            $self.display_decimals = data.display_decimals;
+            $self.display_decimals = data.display_decimals || 2;
```

Comment:
`data.display_decimals` returned `undefined` which assigned the `new_y`, under `formatter`, an int value.